routes/api.php

```diff
@@ -136,6 +136,7 @@
             Route::post('ajax/rebuild-cache', '\Djoudi\LaravelH5p\Http\Controllers\AjaxController@rebuildCache')->name('h5p.ajax.rebuild-cache');
             Route::any('ajax/filter', '\Djoudi\LaravelH5p\Http\Controllers\AjaxController@filter')->name('h5p.ajax.filter');
             Route::any('ajax/finish', '\Djoudi\LaravelH5p\Http\Controllers\AjaxController@finish')->name('h5p.ajax.finish');
+            Route::any('mobile-app-ajax/finish', '\Djoudi\LaravelH5p\Http\Controllers\MobileAppAjaxController@finish')->name('h5p.ajax.finish');
```

Comment:
We call mobile as Curriki Reader. can you please make endpoint as:  'ajax/reader/finish'